web/styles/zulip.css

```diff
@@ -1145,6 +1145,10 @@ strong {
     word-wrap: break-word;
 }
 
+.message_view_header_tooltip {
+    word-wrap: break-word;
```

Comment:
I'm pretty sure we have word wrapping enabled in all tooltips now, so I believe this line is redundant. Could you verify that?

Reply:
Yup, I didn't saw it earlier, my bad. Removed this part.